webapp/web/app.yaml

```diff
@@ -10,6 +10,14 @@ inbound_services:
 
 default_expiration: "1d"
 
+vpc_access_connector:
```

Comment:
This is more of a personal question: a VPC network is not needed anywhere else in this project? e.g. datastore?

Reply:
No. It *seems* to me that calling a Google Cloud API generally doesn't require the VPC network, but connecting directly to an internal IP from Serverless (including App Engine) does.

Also there's actually a problem here. The prod project has a different name for its VPC access connector... Let me see how I can fix that

Reply:
Yea I was gonna mention that the prod and staging share the same `env_variables` below while vpc_access_connector is different from prod and staging (both names and regions). 

Also I assume there shouldn't an issue if prod and staging share the same redis? e.g. caching to the same key name? Does it mean that they share 1GB together?

Reply:
So the two projects are completely separate. They in fact have their own internal network. And the Redis instances just happen to share the same IP (on two different private networks) between them.

The VPC connector is less fortunate, as it has the project and location ID coded in as you said.